### ISSUE TITLE
Add empty builder signatures

### ISSUE THREAD

**Issue by Dimava** [ecyrbe/zodios#490]
Currently I have to 
```ts
export const contract = apiBuilder({
  method: 'get',
  path: '/ping',
  alias: 'ping',
  response: z.literal('pong'),
})
  .addEndpoint({
    method: 'get',
    path: '/ping',
    alias: 'ping',
    response: z.literal('pong'),
  })
  .build()
```
causing possible not-pretty diffs anf indentation
I would like to be able to
```ts
export const contract = apiBuilder()
  .addEndpoint({
    method: 'get',
    path: '/ping',
    alias: 'ping',
    response: z.literal('pong'),
  })
  .build()
```
instead

**Comment by ecyrbe** [comment]
this should be doable without breaking change.

**Comment by Dimava** [comment]
@ecyrbe ?

**Comment by ecyrbe** [comment]
sorry, was closed automatically. it's still on todo

**Comment by ecyrbe** [comment]
@Dimava this is now available in `10.9.6`
